### PR TITLE
Israel new fetcher lambda

### LINKED ISSUE #483: New data source for Israel
We've got a new, API-based data source for Israel. An API key is needed for access which we have and I can provide on request if anyone is interested in trying to develop this. I believe this is the same data we're already getting for IL, so we'll want to deprecate the old source when adding this one.

- [List of available commands](https://github.com/openaq/openaq-fetch/files/1857877/GETcommands.docx)
- [Overview of available data](https://github.com/openaq/openaq-fetch/files/1857886/IsraelNetwork-Eng.xlsx)

### LINKED ISSUE #930: Israel Provider
## Overview 
refer to the [wiki](https://github.com/openaq/openaq-fetch/wiki/Israel-Ministry-of-Environmental-Protection)

Adapter is written and under testing

### PR DESCRIPTION
**Pull request by majesticio**
closes #483 
closes #930 
Adds 182 stations

### CODE REVIEW

File: src/adapters/envista.js [hunk 1]
```diff
@@ -1,482 +1,167 @@
...
+import pkg from 'lodash';
+import { parallel, parallelLimit } from 'async';
+import { convertUnits, unifyMeasurementUnits } from '../lib/utils.js';
+
+const headers = {
+  'User-Agent': 'Mozilla/5.0 (X11; Ubuntu; Linux x86_64; rv:54.0) Gecko/20100101 Firefox/54.0',
+  'Accept': 'text/html,application/json,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
+  'Content-Type': 'text/html; charset=utf-8',
+  'envi-data-source': 'MANA',
+  'Authorization': 'ApiToken ' + '1cab20bf-0248-493d-aedc-27aa94445d15'
```
Comment (russbiggs) [inline src/adapters/envista.js:16]: pull the api key out as a secret so its just a variable in the file

File: src/adapters/envista.js [hunk 2]
```diff
@@ -1,482 +1,167 @@
...
-  return function (done) {
-    request(source)(link, (err, res, body) => {
-      if (err || res.statusCode !== 200) {
-        return done(null, []);
-      }
 
-      const $ = cheerio.load(body);
-      if (!hasAcceptedParameters($)) {
-        return done(null, []);
+    parallelLimit(tasks, 16, (err, results) => { //  Magic number to avoid rate limiting is 16.
```
Comment (russbiggs) [inline src/adapters/envista.js:62]: Does 17 break it? would going down to 15 break anything else. Might be better safe than sorry

File: src/sources/il.json [hunk 3]
```diff
@@ -1,17 +1,17 @@
 [
     {
-        "url": "http://www.svivaaqm.net/",
+        "url": "https://api.svivaaqm.net/v1/envista/",
         "adapter": "envista",
-        "name": "Israel",
+        "name": "Israel Envista",
```
Comment (russbiggs) [inline src/sources/il.json:5]: let change this so it follows the "key" style (sorry) so something like `israel-envista`

**Review by russbiggs — CHANGES_REQUESTED** [review]

File: src/adapters/envista.js (hunk 2 above)
Comment (majesticio) [reply src/adapters/envista.js:62]: I'll reduce it to 15 to be safe 👍🏼

File: src/adapters/envista.js (hunk 1 above)
Comment (russbiggs) [reply src/adapters/envista.js:16]: use environment variables like we are in the other adapters e.g. https://github.com/openaq/openaq-fetch/blob/508287ac720f06dbc0ed1295ab8e2e8c76dc4fd5/adapters/au_vic.js#L31 then we'll place is in the secrets location.

Comment (majesticio) [reply src/adapters/envista.js]: Ok it is set as `process.env.ISRAEL_ENVISTA_TOKEN`

**Pull request merged**